packages/plans-grid/src/plans-table/plan-item.tsx

```diff
@@ -96,9 +108,19 @@ const PlanItem: React.FunctionComponent< Props > = ( {
 						}
 						className="plan-item__summary"
 					>
-						<div className="plan-item__heading">
+						<div
+							className={ classNames( 'plan-item__heading', {
+								'badge-next-to-name': popularBadgeVariation === 'NEXT_TO_NAME',
+							} ) }
+						>
 							<div className="plan-item__name">{ name }</div>
+							{ isPopular && popularBadgeVariation === 'NEXT_TO_NAME' && (
```

Comment:
If we export the `popularBadgeVariation ` values, we can use them here instead of hard coding `'NEXT_TO_NAME'`

Reply:
Same as above.